SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -49,30 +49,30 @@ <h2>Question by {{ currentQuestion.owner }} on {{ currentQuestion.creation_date
   </div>
   <div id="answers">
   <!-- Display the answers -->
-  {% for answer in bestAnswer %}
+  {% if bestAnswer %}
       <form id="deselect{{ answer.id }}_form" method="post">
```

Comment:
I think here you need to use `{{ bestAnswer.id }} `as well instead of `{{ answer.id }}`.

Reply:
actually, you can just get rid of {{ answer.id }} entirely since id isn't used for anything in particular. Even if it is, it can just be left as deselect since there is always only one instance of best answer.

Reply:
Very true.